android/src/main/java/com/getcapacitor/plugin/http/CapacitorCookieManager.java

```
@@ -141,20 +140,18 @@ public void removeAllCookies() {
     }
 
     /**
-     * Removes all cookies for the given url. This method is asynchronous.
-     *
+     * Removes all cookies for the given URL. NOT YET IMPLEMENTED - internally invokes
```

I would raise an exception with this message (not yet implemented) instead of documenting it. But the method is currently not used, hopefully we remember it later on.

---

I reject the call in Http.java
But add the exception here as well.